apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -56,10 +57,11 @@ export const MultiFactorAuthentication: React.FunctionComponent<MfaProps> = (pro
     } = props;
 
     const { t } = useTranslation();
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
No need to have any here. `Dispatch` internally adds `AnyAction`.

```suggestion
    const dispatch: Dispatch = useDispatch();
```